docs/upgrade-notes/upgrade-rds.md

```diff
@@ -0,0 +1,246 @@
+
+- `${PREFIX}-data-migration2` Lambda
+- `${PREFIX}-postgres-migration-async-operation` Lambda
+
+### 5. Run the second data migration
+
+Now that Cumulus module is deployed, we can use some newly created resources to migrate granule, execution, and PDR data from DynamoDB to our PostgreSQL database.
+
+This second data migration process can be run by invoking the provided `${PREFIX}-postgres-migration-async-operation` Lambda included in the Cumulus module deployment.
+This Lambda starts an asynchronous operation which runs as an ECS task to run the migration.
+
+To invoke the Lambda and start the data migration, you can use the AWS Console or CLI:
+
+```bash
+aws lambda invoke --function-name $PREFIX-postgres-migration-count-tool \
+  --payload $PAYLOAD $OUTFILE
+```
+
+where
+
+- `PAYLOAD` is a base64 encoded JSON object. For example:
+
+    ```bash
+    --payload $(echo '{"executionMigrationParams": { "parallelScanSegments": 50,
+    "writeConcurrency": 50 }}' | base64)
+    ```
+
+- `OUTFILE` is the filepath to store the output from the Lambda.
+- `PREFIX` is your Cumulus deployment prefix.
+
+The Lambda will trigger an Async Operation and return an `id` such as:
+
+```json
+{"id":"7ccaed31-756b-40bb-855d-e5e6d00dc4b3","status":"RUNNING",
+"taskArn":"arn:aws:ecs:us-east-1:AWSID:task/$PREFIX-CumulusECSCluster/123456789",
+"description":"Data Migration 2 Lambda ECS Run","operationType":"Data Migration"}
+```
+
+which you can then query the Async Operations [API Endpoint](https://nasa.github.io/cumulus-api/#retrieve-async-operation) for the output or status of your request. If you want to directly observe the progress of the migration as it runs, you can view the CloudWatch logs for your async operations (e.g. `PREFIX-AsyncOperationEcsLogs`).
+
+Since this data migration is copying **all of your execution, granule, and PDR data from DynamoDB to PostgreSQL**, it can take multiple hours (or even days) to run, depending on how much data you have and how much parallelism you configure the migration to use. In general, the more parallelism you configure the migration to use, the faster it will go, **but the higher load it will put on your PostgreSQL database. Excessive database load can cause database outages and result in data loss.** Thus, the parallelism settings for the migration are intentionally set by default to conservative values but are configurable.
+
+See the description of payload parameters below for how to configure the parallelism of the migration.
+
+For reference, using a value of `50` for `executionMigrationParams.parallelScanSegments` and `50` for `executionMigrationParams.writeConcurrency` migrated ~900,000 execution records in 45 minutes and did not spike Aurora PostgreSQL database capacity above 2 ACUs. Similar speed and database capacity usage was observed when using `50` for both the `granuleMigrationParams.parallelScanSegments` and `granuleMigrationParams.writeConcurrency` in testing for the granules migration.
```

Comment:
Yes, they are setting it for each data type, because the number of records for each data type could be very different. 

None of this config is required because there are defaults, but the defaults wouldn't result in a very quick data migration vs the values I used for testing against PODAAC data. I debated just updating the defaults to what I used for PODAAC, but that could be too aggressive depending on your database setup

Reply:
Move this section up to where we discuss the payload being optinoal